test/yaml-rest-runner/src/test/java/org/elasticsearch/test/rest/yaml/section/DoSectionTests.java

```diff
@@ -626,6 +629,28 @@ public void testNodeSelectorByVersion() throws IOException {
         }
     }
 
+    public void testNodeSelectorByVersionRangeFailsWithNonSemanticVersion() throws IOException {
```

Comment:
How about a test of "current"?

Reply:
"current" was already there, I simply changed it in a way that is compatible but does not use Version.CURRENT; a test `testNodeSelectorCurrentVersion` was already there (and it's still passing).

Reply:
Ah thanks, couldn't see it without expanding :)